### ISSUE TITLE
CTK 5.2.1 : [ Error ] ModuleNotFoundError : No module named 'packaging'

### ISSUE THREAD

**Issue by YoussefEssalhi** [TomSchimansky/CustomTkinter#2036]
I encountered this following error while running my Python script with **CTK** Version **5.2.1** :

```cmd
Traceback (most recent call last):
  File "c:\Users\YoussefEssalhi\GUI\main.py", line 4, in <module>
    from customtkinter import CTkLabel, CTkEntry, CTkButton, CTkCheckBox
  File "c:\Users\YoussefEssalhi\GUI\Env\lib\site-packages\customtkinter\__init__.py", line 10, in <module>
    from .windows.widgets.appearance_mode import AppearanceModeTracker
  File "c:\Users\YoussefEssalhi\GUI\Env\lib\site-packages\customtkinter\windows\__init__.py", line 1, in <module>
    from .ctk_tk import CTk
  File "c:\Users\YoussefEssalhi\GUI\Env\lib\site-packages\customtkinter\windows\ctk_tk.py", line 7, in <module>
    from packaging import version
ModuleNotFoundError: No module named 'packaging'
```

**- Expected :** The Script should run without any import errors related to the packaging module.

**- Actually :** The ModuleNotFoundError occurs because the packaging module is missing.

**- Solution :** To resolve this issue, you can install the packaging module using **`pip`**.

Open your terminal or command prompt and enter the following command : `pip install packaging` .

**Comment by davidmarcelbaum** [comment]
I can confirm this behavior after moving from version 5.2.**0** to 5.2.**1**.

**Comment by FRANK0BAN** [comment]
@YoussefEssalhi

I also had this error. All you need to do is change the library to an **older version**, e.g: 5.0.2 _(I use at least that one, but you can use a newer one if it works fine for you)_. 

If you are programming in **PyCharm** and want to change the library versions you can do it like this:

1. enter your project:
> 2.  File>Settings>Project: YourProjectName>PythonInterpreter>customtkinter

3. you must now check the options: "Specify version".
4. select the version on which you are not getting the error.

I hope I helped.

**Comment by evokerking1** [comment]
Found how to fix it you have to install packaging

**Comment by renger08** [comment]
just use `pip install packaging`
its worked for me

**Comment by TomSchimansky** [comment]
Should be fixed with 5.2.2, as packaging is now added to requirements
